vips/vip-551/bsctestnet.ts

```diff
@@ -0,0 +1,286 @@
+export const EMODE_POOL_SPECS = {
+  label: "Stablecoins",
+  id: 1,
+  markets: [vUSDT, vUSDC, vPT_USDe_30Oct2025],
+  marketsConfig: [
+    {
+      address: vUSDT,
+      collateralFactor: parseUnits("0", 18),
+      liquidationThreshold: parseUnits("0", 18),
+      liquidationIncentive: parseUnits("1.08", 18),
+      borrowAllowed: true,
+    },
+    {
+      address: vUSDC,
+      collateralFactor: parseUnits("0.", 18),
+      liquidationThreshold: parseUnits("0", 18),
+      liquidationIncentive: parseUnits("1.08", 18),
+      borrowAllowed: true,
+    },
+    {
+      address: vPT_USDe_30Oct2025,
+      collateralFactor: parseUnits("0.90", 18),
+      liquidationThreshold: parseUnits("0.92", 18),
+      liquidationIncentive: parseUnits("1.08", 18),
+      borrowAllowed: false,
+    },
+  ],
+};
+
+export const convertAmountToVTokens = (amount: BigNumber, exchangeRate: BigNumber) => {
+  const EXP_SCALE = parseUnits("1", 18);
+  return amount.mul(EXP_SCALE).div(exchangeRate);
+};
+
+const vTokensMinted = convertAmountToVTokens(marketSpecs.initialSupply.amount, marketSpecs.vToken.exchangeRate);
+const vTokensRemaining = vTokensMinted.sub(marketSpecs.initialSupply.vTokensToBurn);
+
+const configureConverters = (fromAssets: string[], incentive: BigNumberish = CONVERSION_INCENTIVE) => {
+  enum ConversionAccessibility {
+    NONE = 0,
+    ALL = 1,
+    ONLY_FOR_CONVERTERS = 2,
+    ONLY_FOR_USERS = 3,
+  }
+
+  return Object.entries(converterBaseAssets).map(([converter, baseAsset]: [string, string]) => {
+    const conversionConfigs = fromAssets.map(() => [incentive, ConversionAccessibility.ALL]);
+    return {
+      target: converter,
+      signature: "setConversionConfigs(address,address[],(uint256,uint8)[])",
+      params: [baseAsset, fromAssets, conversionConfigs],
+    };
+  });
+};
+
+export const vip551 = () => {
+  const meta = {
+    version: "v2",
+    title: "VIP-551 [BNB Chain] Add PT-USDe-30Oct2025 markets to the Core pool",
+    description: "VIP-551 [BNB Chain] Add PT-USDe-30Oct2025 markets to the Core pool",
+    forDescription: "I agree that Venus Protocol should proceed with this proposal",
+    againstDescription: "I do not think that Venus Protocol should proceed with this proposal",
+    abstainDescription: "I am indifferent to whether Venus Protocol proceeds or not",
+  };
+
+  return makeProposal(
+    [
+      {
+        target: MOCK_PENDLE_PT_ORACLE,
+        signature: "setPtToSyRate(address,uint32,uint256)",
+        params: ["0x0000000000000000000000000000000000000003", TWAP_DURATION, parseUnits("1", 18)],
+      },
+      {
+        target: bsctestnet.RESILIENT_ORACLE,
+        signature: "setTokenConfig((address,address[3],bool[3],bool))",
+        params: [
+          [
+            PT_USDe_30Oct2025,
+            [PT_USDe_PENDLE_ORACLE, ethers.constants.AddressZero, ethers.constants.AddressZero],
+            [true, false, false],
+            false,
+          ],
+        ],
+      },
+
```

Comment:
We should also configure the parameters related to the Capped Oracle. See [this](https://github.com/VenusProtocol/vips/pull/587/files#diff-f04a599c3ab00b9f0421abe892758b7c800ee84d8f7a02fa59a28ce484022b94R215-R235):
* setSnapshot(uint256,uint256)
* setGrowthRate(uint256,uint256)
* setSnapshotGap(uint256)